charts/common/templates/_service.yaml

```
@@ -0,0 +1,28 @@
+{{/* 
+Create a Service
+
+Usage:
+{{ include "common.service" . }}
+
+Params:
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 2 but found 0

(indentation)